clientv1/defaultplugins/grpcadapter/data_resync_grpc.go

```diff
@@ -19,13 +19,14 @@ import (
 
 	"github.com/ligato/cn-infra/db/keyval"
 	"github.com/ligato/vpp-agent/clientv1/defaultplugins"
+	"github.com/ligato/vpp-agent/flavors/rpc/model/vppsvc"
 	"github.com/ligato/vpp-agent/plugins/defaultplugins/aclplugin/model/acl"
 	"github.com/ligato/vpp-agent/plugins/defaultplugins/ifplugin/model/bfd"
 	"github.com/ligato/vpp-agent/plugins/defaultplugins/ifplugin/model/interfaces"
 	"github.com/ligato/vpp-agent/plugins/defaultplugins/l2plugin/model/l2"
 	"github.com/ligato/vpp-agent/plugins/defaultplugins/l3plugin/model/l3"
-	"github.com/ligato/vpp-agent/flavors/rpc/model/vppsvc"
 	"golang.org/x/net/context"
+	"golang.org/x/tools/go/gcimporter15/testdata"
```

Comment:
this was probably added by mistake, wasn't it?

Reply:
Exactly. Removed.